index.js

```diff
@@ -45,6 +47,15 @@ let supportLevel = (() => {
 	}
 
 	if (process.platform === 'win32') {
+		// First version of Node.js to include patch to libuv that enables colour
+		// output on Windows. Anything earlier and it won't work.
+		if (semver.lt(process.version, '7.5.0')) {
```

Comment:
Node.js 7 is not LTS, so I would be more comfortable targeting Node.js 8 here. In case of problem, nothing will be fixed in Node.js 7.